.devcontainer/devcontainer.json

```diff
@@ -8,20 +8,24 @@
 		"args": { "NODE_VERSION": "12" }
 	},
 
+    // Invoke 'nvm' to install our preferred version of node, per the '.nvmrc' file
+    // at the root of the ${workspaceFolder}.
+    "postCreateCommand": ". /usr/local/share/nvm/nvm.sh; nvm install",
```

Comment:
Automatically run 'nvm install' when the container is created.  (Otherwise, you'll see prompts when running tests, etc.)